File: Slack.App/ClientApp/src/app/user/chat-message.ts
export class ChatMessage {
  userId: number;
  user: string;
  message: string;
  room: string;

  constructor(userId: number = 0, user: string = '', message: string = '', room: string = '') {
    this.userId = userId;
    this.user = user;
    this.message = message;
    this.room = room;
  }
}

File: Slack.Core/ChatMessage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Slack.Core
{
    public class ChatMessage
    {
        public int userId { get; set; }
        public string user { get; set; }

        public string message { get; set; }

        public string room { get; set; }
    }
}
